packages/winpanda/extra/src/winpanda/tests/unit/test_core_utils.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+import mock
+
+from core import utils
+from pathlib import Path
+from core.exceptions import RCError
+
+
+def test_rc_load_string_should_fail():
+    """Check error validation when path not found."""
+    with pytest.raises(AssertionError):
+        utils.rc_load_json('')
+
+
+@mock.patch('core.utils.Path.is_absolute', return_value=True)
+def test_rc_load_absolute_path_should_fail(*args):
```

Comment:
The path you generate here is relative (`.`) so should fail with an `AssertionError`.

Since you mock the check for an absolute path, it then fails because of a different error (probably an `OSError` since the supposed JSON file is a directory).

Reply:
You are right, that was my fault. Function name and description changed.